.husky/prepare-commit-msg

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+. "$(dirname "$0")/_/husky.sh"
+
+COMMIT_MSG_FILE=$1
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+BRANCH_NUMBER=$(echo "$BRANCH_NAME" | grep -o '[0-9]\+')
+
+if [ ! -z "$BRANCH_NUMBER" ];
+then
+  COMMIT_MSG=$(cat "$COMMIT_MSG_FILE")
+
+  MODIFIED_MSG=$(echo "$COMMIT_MSG" | sed "s/^\(.*: \)\(.*\)/\1$BRANCH_NUMBER - \2/")
+
+  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): [0-9]{1,5} - ([a-zA-Z]+ )"
```

Comment:
I thought it would be something like that with 'test' or without:
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore): \d{1,5} \-( [a-zA-Z0-9]+){2,}$"
```

Reply:
With the exception of the numbers in the description, what difference do you see?

Reply:
UPD. I removed numbers to prevent errors with manual branch number writing, e.g if I write as we did early - 'fix: 224 - some fixes', now commit msg wil be  'fix: 224 - 224 - some fixes'

Reply:
But there is a difference. In the current template I can write 1 Latin letter and then any characters including numbers. Try to name the commit for example 'fix: 224 - a 123', it should match.

Reply:
```suggestion
  COMMIT_REGEX="^(feat|fix|refactor|docs|chore|test): \d{1,5} \-( [a-zA-Z-]+){2,}$"
```

Reply:
Changed